src/main/webapp/app/exam/participate/exercises/text/text-exam-submission.component.html

```diff
@@ -1,17 +1,20 @@
 @if (exercise) {
-    <h3 class="text-align-left fw-normal">
-        <span>
-            {{ exercise.exerciseGroup?.title }}
-        </span>
-        <!-- prettier-ignore -->
-        <span left-header>&nbsp;({{ exercise.maxPoints }} {{ 'artemisApp.examParticipation.points' | artemisTranslate }}@if (exercise.bonusPoints) {
-            <span
-            >, {{ exercise.bonusPoints }} {{ 'artemisApp.examParticipation.bonus' | artemisTranslate }}</span>
+    <div class="d-flex justify-content-between align-items-center">
+        <h3 class="text-align-left fw-normal mb-0">
+            <span>
+                {{ exercise.exerciseGroup?.title }}
+            </span>
+            <!-- prettier-ignore -->
+            <span left-header>&nbsp;({{ exercise.maxPoints }} {{ 'artemisApp.examParticipation.points' | artemisTranslate }}@if (exercise.bonusPoints) {
+                <span
+                >, {{ exercise.bonusPoints }} {{ 'artemisApp.examParticipation.bonus' | artemisTranslate }}</span>
```

Comment:
I understand this is not part of your PR, but could you try to remove the` <!-- prettier-ignore -->`, the odd formatting, and the use of `&nbsp;`? The same applies to the `modeling-exam-submission.component.ts.`